examples/form_inputs/www/shiny-semantic-radiogroup.js

```diff
@@ -0,0 +1,92 @@
+var semanticRadioGroupBinding = new Shiny.InputBinding();
```

Comment:
Don't we want to have it in `inst/www`?

Reply:
Oops...left that file in by accident, was testing it out but can use the `shiny-input-radiogroup` class instead.